.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -13,4 +13,4 @@ dependencies:
   - pip:
     - future
     - scipy
-    - av
+    - av < 10
```

Comment:
Ah good point, I thought this is not a bug on pyav but rather some changes that have BC behaviour, so I thought this was a temporary fix while we will have a long term fixes after knowing what actually changes (then change back and remove this change)

But it might be a bug on pyav side as well as you suggested.

Reply:
My bad for providing to little context here. I've updated the top comment.